optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/DefaultPartitionedSearchPhase.java

```diff
@@ -128,6 +132,7 @@ public void solve(DefaultSolverScope<Solution_> solverScope) {
                         partitionSolver.solve(part);
                         partitionQueue.addFinish(partIndex);
                     } catch (Throwable throwable) {
+                        // TODO is it a good idea to catch Throwables including OOME for example?
```

Comment:
Remove todo, but clarify with a comment:
for example: // Transfer from the part thread to the solver thread, so the solver thread throws an exception